docs/codeql/writing-codeql-queries/about-codeql-queries.rst

```diff
@@ -114,6 +116,8 @@ You can modify the alert message defined in the final column of the ``select`` s
 
 Select clauses for path queries (``@kind path-problem``) are crafted to display both an alert and the source and sink of an associated path graph. For more information, see ":doc:`Creating path queries <creating-path-queries>`."
 
+Select clauses for diagnostic queries (``@kind diagnostic``) and summary metric queries (``@kind metric`` and ``@tags summary``) have different requirements. For examples, see the `diagnostic queries <https://github.com/github/codeql/search?q=%22%40kind+diagnostic%22>`__ and the `summary metric queries <https://github.com/github/codeql/search?q=%22%40kind+metric%22+%22%40tags+summary%22>`__  in the CodeQL repository.
```

Comment:
@adityasharad - I wrote this without knowing what the requirements were. You kindly pointed me to https://github.com/github/codeql-coreql-team/issues/816 and https://github.com/github/codeql-coreql-team/issues/817. Shall we update this section to be more explicit or are you happy to leave this with just the links to examples for now?